config/locales/rs.yml

```diff
@@ -515,29 +515,29 @@ rs:
       our_events: Naši događaji
       our_partners: Naši partneri
     about:
-      description: S95 je otvoren platform koji objedinjuje nezavisne park trke, koje se održavaju prema tri postulata - subota, 9 ujutru, 5 km.
+      description: S95 je otvorena platforma koja objedinjuje nezavisne park trke koje se održavaju prema tri postulata - subota, 9 ujutru, 5 km.
       events: 'Sistem trenutno prikazuje rezultate %{events_count} lokacija: %{events}.'
       join_us:
-        title: Želite da se pridružite platform S95?
+        title: Želite da se pridružite platformi S95?
         body_html: >
-          <p>In short, there is needed your desire.</p>
+          <p>Ukratko, potrebna je samo volja.</p>
           <p>
-            In more detail, you need:
+            Detaljnije, potrebni su:
             <ul>
-              <li>safe and comfortable route</li>
-              <li>a team that ready to handle runs every saturday all year round</li>
-              <li>a desire to make everyone healthier and happier</li>
+              <li>bezbedna i stazasafe and comfortable route</li>
```

Comment:
Maybe it would be better to completely translate this phrase to serbian without mix?

Reply:
Yes, missed that, I guess comfortable meant that it easy to get to / access so I'll translate it that way.

Reply:
Yes, missed that, I guess comfortable meant that it easy to get to / access so I'll translate it that way.